src/ansys/fluent/core/quantity.py

```diff
@@ -376,7 +376,7 @@ def get_si_unit_from_dim(dim_list):
         "M": "kg",
         "L": "m",
         "T": "s",
-        "Temp": "K",
+        "?": "K",
```

Comment:
@dnwillia-work @seanpearsonuk @hpohekar One issue with using characters such as these or even "mu" is that they are not readily accessible from the keyboard and one will have to copy them from some place.

Reply:
Yeah, it's true.  I was struggling with this yesterday.  I would say it's only an issue if the characters leak through the API and become part of the interface.   If it's just internal implementation, then I'm ok with it.

Reply:
Here's a commit where I fixed up the question marks.

https://github.com/pyansys/pyfluent/commit/f10020b9c78ad368c18599226e58760636acd3d9

It seems OK.  I think it's important to do this in an editor where the encoding is UTF-8.

Reply:
The issue is not whether it appears OK in editors, but how users will enter such a symbol in the first place as it is not readily available in the keyboard.

Reply:
Yes, got it, hence my first comment where I said "it's only an issue if the characters leak through the API ..." .  They do not leak through right now but, that said, probably it's best to stick to keyboard only characters.

Reply:
@dnwillia-work @h-krishnan @seanpearsonuk This is resolved now.

The important point is that both `C++ units library` and `pint` uses `u` for `µ`.

So we have replaced `µ` with `u`.

`Theta` and `ohm` sign replaced by `Temp` and `SAngle` respectively.